apps/super-app/src/routes/SupportDetailPage.tsx

```diff
@@ -117,29 +139,33 @@ function DescriptionSection(props: DescriptionProps) {
 
 type CtaProps = Pick<
   Twc2SupportDetail,
-  'ctaBlurb' | 'ctaButtonText' | 'ctaLink' | 'whatsappLogoSrc'
+  'ctaButtonText' | 'ctaLink' | 'ctaIcon'
 >;
 
 function CtaSection(props: CtaProps) {
-  const { ctaBlurb, ctaButtonText, ctaLink, whatsappLogoSrc } = props;
+  const { ctaButtonText, ctaLink, ctaIcon } = props;
+  // TODO: https://github.com/bettersg/call-home/pull/144#discussion_r1141414590
```

Comment:
If possible, do leave your own interpretation of the conversation so that a) readers don't have to make a round trip to reread the conversation b) readers know exactly what you're referencing instead of reading the whole conversation to figure it out.

In this case, I'm not sure what you're referencing because you've already implemented my suggestion here:

> If we want a generic CTA that can accept any image URL, then sure, parameterising is fine

by accepting the `ctaIcon` prop, so I don't think there's anything left to do.